docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md

```diff
@@ -0,0 +1,273 @@
+
+```azurecli
+az acr scope-map create \
+  --description "Nested connected registry repo pull scope map." \
+  --name nested-connected-registry-pull \
+  --registry mycontainerregistry001 \
+  --repository "acr/connected-registry" content/read
+```
+
+Next, use the following command to create a client token for the IoT Edge device and associate it to the scope map:
+
+```azurecli
+az acr token create \
+  --name nestedcrimagepulltoken \
+  --registry mycontainerregistry001 \
+  --scope-map connected-registry-pull
+```
+
+This command will print a JSON that will include credential information similar to the following:
+
+```json
+  ...
+  "credentials": {
+    "activeDirectoryObject": null,
+    "certificates": [],
+    "passwords": [
+      {
+        "creationTime": "2020-12-10T00:06:15.356846+00:00",
+        "expiry": null,
+        "name": "password1",
+        "value": "$$$0meCoMPL3xP4$$W0rd001!@#$$"
+      },
+      {
+        "creationTime": "2020-12-10T00:06:15.356846+00:00",
+        "expiry": null,
+        "name": "password2",
+        "value": "#$an0TH3rCoMPL3xP4ssW0rd002!#$"
+      }
+    ],
+    "username": "nestedcrimagepulltoken"
+  }
+  ...
+```
+
+You will need the `username` and one of the `passwords` values for the IoT Edge manifest below.
+
+  > [!IMPORTANT]
+  > Make sure that you save the generated passwords. Those are one-time passwords and cannot be retrieved. You can generate new passwords using the [az acr token credential generate][az-acr-token-credential-generate] command.
+
+More details about tokens and scope maps are available in [Create a token with repository-scoped permissions](container-registry-repository-scoped-permissions.md).
+
+Last, add the client token to the top level connected registry using the following command:
+
+```azurecli
+az acr connected-registry update \
+  --name myconnectedregistry \
+  --registry mycontainerregistry001 \
+  --add-client-tokens nestedcrimagepulltoken
+```
+  > [!IMPORTANT]
+  > The client token must be added to the parent connected registry, `myconnectedregistry` in this case.
+
+## Retrieve connected registry configuration information
+
+Before deploying the connected registry to the nested IoT Edge device, you will need to retrieve the configuration from the connected registry resource in Azure. Use the [az acr connected-registry install][az-acr-connected-registry-install] command to retrieve the configuration.
+
+```azurecli
+az acr connected-registry install renew-credentials \
+  --registry mycontainerregistry001 \
+  --name myconnectedmirror \
+```
+
+This will return the connection string for the connected registry including the newly generated passwords.
+
+```json
+{
+  "ACR_REGISTRY_CONNECTION_STRING": "ConnectedRegistryName=myconnectedmirror;SyncTokenName=myconnectedmirror-sync-token;SyncTokenPassword=s0meCoMPL3xP4$$W0rd001!@#;ParentGatewayEndpoint=mycontainerregistry001.westus2.data.azurecr.io;ParentEndpointProtocol=https",
```

Comment:
`ParentGatewayEndpoint=mycontainerregistry001.westus2.data.azurecr.io;ParentEndpointProtocol=https` needs to be the parent connected registry (`myconnectedregistry`) endpoint

Reply:
Actually, this should be what is returned from the CLI and it will include placeholders only.